samples/build.gradle

```diff
@@ -1,4 +1,7 @@
 apply plugin: 'org.jetbrains.kotlin.multiplatform'
+apply plugin: 'application'
+
+mainClassName = System.getProperty("mainClass")
```

Comment:
Unless I'm missing something, the `application` plugin doesn't have a built in way to pass in the main class from the CLI, this line makes it possible to do:
```
./gradlew -DmainClass=okio.samples.Test run
```

Reply:
I'd just been running these from IntelliJ.

Reply:
That didn't work for me for some reason, nothing happens when I tap the Run button.